src/sentry/static/sentry/app/views/settings/components/forms/tableField.tsx

```diff
@@ -116,7 +116,9 @@ export default class TableField extends React.Component<Props> {
           <Alert type="error">
             <span
               dangerouslySetInnerHTML={{
-                __html: singleLineRenderer(confirmDeleteMessage || ''),
+                __html: singleLineRenderer(
+                  confirmDeleteMessage || 'Are you sure you want to delete this item?'
```

Comment:
Should this have a t() around it?

Reply:
@markstory We can definitely translate the default text but not the configurable one